src/Core/StringEx.cs

```diff
@@ -0,0 +1,44 @@
+﻿#region License
+/* 
+ * Copyright (C) 1999-2018 Pavel Tomin.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2, or (at your option)
+ * any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; see the file COPYING.  If not, write to
+ * the Free Software Foundation, 675 Mass Ave, Cambridge, MA 02139, USA.
+ */
+#endregion
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Reko.Core
+{
```

Comment:
Consider inlining this in the class that is using it. Only if we find that we are splitting into lines in many places does it make sense to create an extension class for `System.String`.

Reply:
I'm going to use it at `CodeFormatter` also

Reply:
I'm still unhappy about tacking this onto the `string` class. Consider using it inlining it in the two places you're using it.

Reply:
OK